gloo/transport/uv/device.cc

```diff
@@ -108,6 +108,15 @@ static bool lookupAddrForIface(struct attr* attr) {
 }
 
 static void lookupAddrForHostname(struct attr& attr) {
+#ifdef _WIN32
+  WSADATA wsa_data;
```

Comment:
Since the rest of the team have limited experiences on Windows, shall we add some comments/pointers to explain this?

Reply:
sure, will do.

Reply:
Good old WSAStartup 🎺 
Don't we want/need to guard initialization block with `std::call_once`?
And it looks like the same is needed for tcp transport?

Reply:
I do not enable tcp tranport on Windows, so currently we do not need do the same to tcp transport. Base on MSDN doc, we can call WSAStartup multiple times, it only loads DLL for first time call, for other calls, just increment a ref count. But I still guard it with std::call_once.